lib/EventHandler.js

```diff
@@ -47,6 +55,12 @@ module.exports = class EventHandler {
     };
     this._wm = new watchman(opt, (data) => this.eventHandler(data));
     this._wm.watch();
+    if(this._livenessInterval) {
+      touch.sync('/tmp/liveness');
+      setInterval(() => {
+        touch.sync('/tmp/liveness');
```

Comment:
if(this._wm.watching){
  touch.sync('/tmp/liveness');
}

Reply:
done